README.md

```diff
@@ -76,12 +76,12 @@ As well as the Linux platforms supported, OMI is also tested to work on the foll
 
 - For RPM based systems (RedHat, Oracle, CentOS, SuSE):
 ```
-sudo rpm -Uvh ./omi-1.4.1-0.ssl_100.ulinux.x64.rpm
+sudo rpm -Uvh ./omi-*.*.*-*.ssl_100.ulinux.x64.rpm
```

Comment:
Again, these were supposed to be "general ideas", not "cut and paste" examples. How about doing something like:

sudo rpm -Uvh ./omi-MAJOR.MINOR.PATCH-BUILD.ssl_100.ulinux.x64.rpm

where MAJOR, MINOR, PATCH, and BUILD refer to the version number information for the specific version of OMI that you wish to install.

Something along those lines, anyway.